jekyll/_cci2/configuration-reference.md

```diff
@@ -739,10 +739,58 @@ Key | Required | Type | Description
 version | Y | String | Should currently be `2`
 {: class="table table-striped"}
 
-#### **<`workflow_name`>**
+### **<`workflow_name`>**
 
 A unique name for your workflow.
 
+#### **`triggers`**
+Specifies which triggers will cause this workflow to be executed. Default behavior is to trigger the workflow when pushing to a branch. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+triggers | N | String | Should currently be `schedule`.
+{: class="table table-striped"}
+
+##### **`schedule`**
+A workflow may have a `schedule` indicating it runs at a certain time, for example a nightly build that runs every day at 12am UTC:
+
+```
+workflows:
+   version: 2
+   nightly:
+     triggers:
+       - schedule
```

Comment:
This line is not valid YAML afaict. Should it be `type: schedule` or `schedule:` ?

Reply:
Correct, it's a typo.

Reply:
thanks, fixed